ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_position_only.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 ROS2-Control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef ROS2_CONTROL_DEMO_HARDWARE__RRBOT_SYSTEM_POSITION_ONLY_HPP_
+#define ROS2_CONTROL_DEMO_HARDWARE__RRBOT_SYSTEM_POSITION_ONLY_HPP_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "rclcpp/macros.hpp"
+
+#include "hardware_interface/components/base_interface.hpp"
+#include "hardware_interface/components/system_interface.hpp"
+#include "hardware_interface/handle.hpp"
+#include "hardware_interface/hardware_info.hpp"
+#include "hardware_interface/types/hardware_interface_return_values.hpp"
+#include "hardware_interface/types/hardware_interface_status_values.hpp"
+#include "ros2_control_demo_hardware/visibility_control.h"
+
+using hardware_interface::return_type;
+
+namespace ros2_control_demo_hardware
+{
+class RRBotSystemPositionOnlyHardware : public
+  hardware_interface::components::BaseInterface<hardware_interface::components::SystemInterface>
+{
+public:
+  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemPositionOnlyHardware);
+
+  return_type configure(const hardware_interface::HardwareInfo & info) override;
+
+  std::vector<hardware_interface::StateInterface> export_state_interfaces() override;
+
+  std::vector<hardware_interface::CommandInterface> export_command_interfaces() override;
+
+  return_type start() override;
+
+  return_type stop() override;
+
+  return_type read() override;
+
+  return_type write() override;
+
+private:
+  // Dummy parameters
+  double hw_start_sec_, hw_stop_sec_, hw_slowdown_;
```

Comment:
If they are dummy, why they need to be members? Also, please do one variable per line

Reply:
They are example parameters for the RRbot simulation. so I will change the comment.